bot/resources/tags/message-content-intent.md

```diff
@@ -0,0 +1,18 @@
+---
+aliases: ["mcintent, message_content, message_content_intent"]
```

Comment:
Aliases need to be in the same format as Python lists, see [here](https://raw.githubusercontent.com/python-discord/bot/main/bot/resources/tags/f-strings.md) for an example.